qualtran/bloqs/data_loading/select_swap_qrom_test.py

```diff
@@ -33,25 +33,29 @@
             data,
             block_size,
             id=f"{block_size}-data{didx}",
-            marks=pytest.mark.slow if block_size == 3 and didx == 0 else (),
+            marks=pytest.mark.slow if block_size == 2 and didx == 0 else (),
         )
         for didx, data in enumerate([[[1, 2, 3, 4, 5]], [[1, 2, 3], [3, 2, 1]]])
-        for block_size in [None, 1, 2, 3]
+        for block_size in [None, 0, 1]
```

Comment:
Did this get slower?

Reply:
no, the different is that we now specify log_block_size instead of block_size; where `block_size = 2**log_block_size`.